static/firefox/selenoid/Dockerfile

```diff
@@ -8,5 +8,7 @@ COPY entrypoint.sh /
 
 USER selenium
 
+RUN echo "cookie-file = ~/.config/pulse/cookie" >> /etc/pulse/client.conf
```

Comment:
Here  you are creating config file as user `selenium`, whereas for chrome, opera and yandex - under `root`. Shouldn't this behavior be the same?

Reply:
Yes sorry, my bad.  Great catch! Fixed it in 4349064. :pray: